### PR TITLE
Replace query strings with hashes

### PR DESCRIPTION
**Pull request by kevinshieh**

### CODE REVIEW

**Comment by kevinshieh** [comment]
@SamLau95 @negativetwelve

**Comment by kevinshieh** [comment]
for the securityz

File: app/models/semester.rb [hunk 1]
```diff
@@ -24,10 +24,10 @@ def can_be_destroyed?
   end
 
   def self.current_semester
-    where("is_current_semester = 'true'").first
+    where(is_current_semester: true).first
```
Comment (negativetwelve) [inline app/models/semester.rb]: you can use a find_by here too if you want to be super cool haha

**Comment by negativetwelve** [comment]
@kevinshieh ++ for the securityz

**Pull request merged**